### ISSUE TITLE
Logo URL broken

### ISSUE THREAD

**Issue by Filnor** [SOBotics/Higgs#61]
The URL to the Logo (https://home.cern/images/2014/01/simulated-higgs-decays-four-muons) leads to a 404 page and searching for the post with the search feature at home.cern [gives no results](https://home.cern/search/node?keys=higgs&newsformat=All&topic=All&audience=All&date_from=2014-01-01&date_to=2014-01-02&type=All). However, [searching for the image](https://www.google.com/search?num=20&q=bos%C3%B3n+de+higgs+o+part%C3%ADcula+de+dios&tbm=isch&tbs=simg:CAQSlQEJgdnZ_1tcgSfEaiQELEKjU2AQaBAgVCAcMCxCwjKcIGmAKXggDEibDA2SCDI0MlxetA6wD_1BaEDGeTKeAo_1TOVKYYn0DamN6I3qzeFKRowXVIeYN1wucrKX_1VZgyRul4r74KYcAAcl_1C1tDf6Q-OOViYN7EGZL4BDYI95vhw9oIAQMCxCOrv4IGgoKCAgBEgRkJq5YDA&sa=X&ved=0ahUKEwitlqOniODeAhVF6qQKHSOAD08Q2A4IKygB&biw=1920&bih=948#imgrc=VH_pBXzyBTLFuM:) reveals some other, mostly third-party sources.

**Comment by rjrudman** [comment]
Yep, I noticed this before as well. The link is intended as providing attribution, which is where the page used to be. I'm not really sure how to go about this. I'll leave it as a 404 link unless someone finds out a better page to link/provide attribution to

**Comment by Filnor** [comment]
If the link to the original image suffices, here's the link: https://home.cern/sites/home.web.cern.ch/files/image/featured/2014/01/higgs_event_display_1.jpg Found it using wayback machine.
